Content/ContentTypeResolver/SingleMediaSelectionResolver.php

```diff
@@ -41,6 +41,11 @@ public function resolve($data, PropertyInterface $property, string $locale, arra
 
         $content = $this->mediaSelectionResolver->resolve(['ids' => [$data['id']]], $property, $locale, $attributes);
```

Comment:
the display option should be added to the view - and i think also the `MediaSelectionResolver` does not handle this correctly.

We should add it there and return it in the view object.